apps/aehttp/src/aehttp_dispatch_int.erl

```diff
@@ -488,16 +488,12 @@ handle_request('GetAccountBalance', Req, _Context) ->
 handle_request('GetPeers', _Req, _Context) ->
     case application:get_env(aehttp, enable_debug_endpoints, false) of
         true ->
-            Peers =
-                lists:map(
-                    fun({URI, LastSeen}) ->
-                        #{uri => URI,
-                          last_seen => LastSeen}
-                    end,
-                    aec_peers:all()),
+            Peers = aec_peers:all(),
             Blocked = aec_peers:blocked(),
-            {200, [], #{peers => Peers,
-                        blocked => Blocked}};
+            Enc = fun(Xs) -> [ X#{ pubkey := aec_base58c:encode(peer_pubkey, PK) }
+                               || X = #{ pubkey := PK } <- Xs ] end,
```

Comment:
This list comprehension is an implicit filter. I would prefer hardcoding expectation that all peers have pubkey by using lists:map - though this is debatable so not need to change.